analysis_options.yaml

```diff
@@ -0,0 +1,14 @@
+analyzer:
+  strong-mode:
+    implicit-casts: false
+  errors:
+    todo: ignore
+    unused_import: error
+    unused_element: error
+    unused_local_variable: error
+    dead_code: error
+
+linter:
```

Comment:
You did this work and didn't wire up `pkg:pedantic`? SMH...

Reply:
Because one of those was harder to fix... but I guess I should #66